collections/Lat_liturg/MS_Lat_liturg_f_4.xml

```diff
@@ -48,6 +48,25 @@
                      <note>Private prayers and devotions based on the liturgy of Eastertide. Liturgical texts occur throughout the book, often with notation. Some German texts.</note>
                      <textLang mainLang="la" otherLangs="gml" resp="#NP">Latin and Middle Low German</textLang>
                   </msItem>
+                  <msItem xml:id="MS_Lat_liturg_f_4-item1-item1">
+                        <locus from="1v" to="56v">(fols. 1v–56v)</locus>
+                        <rubric>Easter Vigil</rubric>
```

Comment:
not really a rubric (that would be the manuscript heading "in vigilia Paschae" or similar) - could you change here and below to `note`?